packages/create-cloudflare/templates/do-hello-world/js/src/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Welcome to Cloudflare Workers! This is your first Durable Objects application.
+ *
+ * - Run `npm run dev` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your Durable Object in action
+ * - Run `npm run deploy` to publish your application
+ *
+ * Learn more at https://developers.cloudflare.com/durable-objects
+ */
+
+export class MyDurableObject {
+	constructor(state, env) {}
+
+	async fetch(request) {
```

Comment:
"This is what is invoked when you call `.fetch` on the stub for an individual Durable Object.

You need to pass a Request object and return a Response back to the calling Worker."